packages/terra-clinical-item-display/src/terra-dev-site/doc/example/TextStyles.jsx

```diff
@@ -3,7 +3,6 @@ import ItemDisplay from 'terra-clinical-item-display';
 
 const component = () => (
   <React.Fragment>
-    <ItemDisplay text="Default (same as Primary)" />
     <ItemDisplay text="Primary" textStyle="primary" />
```

Comment:
Lets also update this, for those looking at code examples to not think they need to add a textStyle:
```suggestion
    <ItemDisplay text="Primary" />
```

Reply:
Updated